### PR TITLE
Description and README updates

### LINKED ISSUE #13: Update readme to include all placetypes
Needs more words.

Related: #10

### PR DESCRIPTION
**Pull request by stepps00**
Fixes #13 and #10.

Updated the README file with standardized formatting, added all placetypes, and added/updated descriptions for some. I am open to changes and/or better descriptions...

### CODE REVIEW

**Comment by nvkelso** [comment]
This README is hand crafted, but really should be auto-generated (for a rainy day, not this PR). Tracking that in https://github.com/whosonfirst/whosonfirst-placetypes/issues/15.

File: README.md [hunk 1]
```diff
@@ -202,26 +239,23 @@ Marine areas are considered `common_optional`.
...
+Because all place is disputed. And everyone has a name for a place that will offend someone else.
 
 Microhoods are considered `optional`.
 
 ### neighbourhood
 
 `102312319`
 
-Things no one will agree on. Ever.
+Neighbourhoods can represent an area surrounding a "place", a community grouping, or even a physical area. One thing is certain - neighbourhoods are things no one will agree on. Ever.
```
Comment (nvkelso) [inline README.md]: `One thing is certain - neighbourhoods are things no one will completely agree on. Ever.` (add **completely**)

File: README.md [hunk 2]
```diff
@@ -108,71 +131,85 @@ Continents are considered `common`.
...
-Localadmins are considered `common optional`
+Localadmins are considered `common_optional`.
 
 ### locality
 
 `102312317`
 
-Towns and cities, independent of size or population. Things with neighbourhoods,
-basically.
+Towns and cities, independent of size or population. Things with neighbourhoods, basically.
```
Comment (nvkelso) [inline README.md]: NOTE: In some areas `locality` and `localadmin` are used interchangeably.

File: README.md [hunk 3]
```diff
@@ -108,71 +131,85 @@ Continents are considered `common`.
...
+
+Where things meet.
+
+Intersections are considered `optional`.
 
 ### localadmin
 
 `404221409`
 
 In many countries, the lowest level of government. They contain one or more localities (or "populated places") which themselves have no authority. Often but not exclusively found in Europe.
```
Comment (nvkelso) [inline README.md]: NOTE: In some areas `locality` and `localadmin` are used interchangeably.

File: README.md [hunk 4]
```diff
@@ -108,71 +131,85 @@ Continents are considered `common`.
...
-Counties are considered `common optional`.
+This needs a better - that is more abstract - name. Like "region" instead of state, province, whatever... but for counties.
+
+Counties are considered `common_optional`.
 
 ### dependency
 
 `102312313`
 
-It's not a sub-region of a country but rather dependent on a parent country for
```
Comment (nvkelso) [inline README.md]: NOTE: In some areas `country` and `dependency` are used interchangeably.

File: README.md [hunk 5]
```diff
@@ -108,71 +131,85 @@ Continents are considered `common`.
...
 
 Continents are considered `common`.
 
 ### county
 
-This needs a better - that is more abstract - name. Like "region" instead of
-state, province, whatever... but for counties.
+`102312313`
 
-Counties are considered `common optional`.
```
Comment (nvkelso) [inline README.md]: NOTE: In some areas `country` and `dependency` are used interchangeably.

File: README.md [hunk 6]
```diff
@@ -108,71 +131,85 @@ Continents are considered `common`.
 
 `102312307`
 
-Basically places that issue passports, notwithstanding the details (like
-empires which actually issue the passports...)
+Basically places that issue passports, notwithstanding the details (like empires which actually issue the passports...)
```
Comment (nvkelso) [inline README.md]: NOTE: In some areas `country` and `dependency` are used interchangeably.

Also... country and county should be a-z sorted.

**Comment by thisisaaronland** [comment]
Seems fine to me

File: README.md (hunk 4 above)
Comment (nvkelso) [reply README.md]: Seems like every implementation of WOF gets it confused that both dependency and country are needed (country alone won’t suffice). Is there a better way of calling that out?

**Pull request merged**